src/components/TodoList.js

```diff
@@ -5,7 +5,7 @@ import PropTypes from 'prop-types'
 function TodoList({ todoList, onRemoveTodo }) {
 
     return (
-        <ul>
+        <ul className='todolist'>
             {todoList.map(function (todo) {
```

Comment:
prop `todoList` is not marked as "required" in `propTypes`, so this line could throw an error at run-time if the value is undefined